FAQ: update slow log, DDL and Syncer related questions (#854)
pingcap/docs: FAQ.md

- Put the `syncer.meta` file in a relatively secure disk. For example, use disks with RAID 1.
- Restore the location information of history synchronization according to the monitoring data that Syncer reports to Prometheus regularly. But the location information might be inaccurate due to the delay when a large amount of data is synchronized.

##### If the downstream TiDB data is not consistent with the MySQL data during Syncer's synchronization process, will DML operations cause exits?

> - If data exists in the upstream MySQL and doesn't exist in the downstream TiDB, and the upstream MySQL performs the `UPDATE` or `DELETE` operation on this piece of data, it will not cause errors or exits and the data will not exist during Syncer's synchronization process.

Reviewer: ```suggestion
- If the data exists in the upstream MySQL but does not exist in the downstream TiDB, when the upstream MySQL performs the `UPDATE` or `DELETE` operation on this row of data, Syncer will not report an error and the synchronization process will not exit, and this row of data does not exist in the downstream.
```

> - If conflicts exist in the primary key indexes or the unique indexes in the downstream, preforming `UPDATE` will cause exits while performing `INSERT` will not.

Reviewer: ```suggestion
- If a conflict exists in the primary key indexes or the unique indexes in the downstream, performing the `UPDATE` operation will cause an exit while performing the `INSERT` operation will not cause an exit.
```
